.github/workflows/docker-publish.yml

```diff
@@ -71,35 +72,60 @@ jobs:
             TAGS="${IMAGE}:nightly"
           fi
           echo ::set-output name=tags::${TAGS}
+      -
+        name: Docker meta
+        id: meta-default
+        uses: docker/metadata-action@v5
+        if: matrix.python_images == env.default_python
+        with:
+          images: |
+            ${{ secrets.DOCKER_SOPEL_IMAGE_NAME }}
+            ghcr.io/${{ github.actor }}/sopel
```

Comment:
Won't the `actor` here and on line R99 be whoever pushed or opened the PR, etc.? I would expect `github.repository_owner` instead.

Reply:
eh, you're probably right. I'll update it.